### PR TITLE
Add the AndroidX migration

### PR DESCRIPTION
**Pull request by mattleibow**

### CODE REVIEW

File: src/Android/Hanselman.Android.csproj [hunk 1]
```diff
@@ -213,6 +213,15 @@
       <Version>4.1.0.581479</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup Condition="$([System.Version]::Parse($(XamarinAndroidVersion.Split('-')[0]))) &gt;= $([System.Version]::Parse('9.4.0'))">
```
Comment (jamesmontemagno) [inline src/Android/Hanselman.Android.csproj]: THAT IS FANCY!

Comment (mattleibow) [reply src/Android/Hanselman.Android.csproj]: I just saw that VS2017 - and the others ignore this condition... Maybe I am doing something wrong... It works on the command line.

**Comment by jamesmontemagno** [comment]
Interesting build results @mattleibow  https://dev.azure.com/jamesmontemagno/Hanselman.Forms/_build/results?buildId=1381&view=logs&jobId=cfa20e98-6997-523c-4233-f0a7302c929f&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=fc1efd11-04e3-530f-bdaf-f23130eb71db

I think we need to upgrade the proguard file perhaps?

**Comment by mattleibow** [comment]
Interesting... I see the warning:
```
R8 : warning : The rule `-keep public class * extends androidx.versionedparcelable.VersionedParcelable {
    <init>();
  }` uses extends but actually matches implements.
```

That is weird, I don't think they changed the type from a class to an interface - and anyway, we are using jetifier to migrate the proguard. I'll have a look.

Both are interfaces in the NuGets:
 - https://www.fuget.org/packages/Xamarin.AndroidX.VersionedParcelable/1.0.0-preview02/lib/monoandroid90/Xamarin.AndroidX.VersionedParcelable.dll/AndroidX.VersionedParcelable/IVersionedParcelable
 - https://www.fuget.org/packages/Xamarin.Android.Support.VersionedParcelable/28.0.0/lib/monoandroid90/Xamarin.Android.Support.VersionedParcelable.dll/AndroidX.VersionedParcelable/IVersionedParcelable


The failure is a result of the jarsigner failing. Not sure what that is.

**Pull request closed without merging**